### PR TITLE
Fixes#604 Enhance code quality

### PR DESCRIPTION
**Pull request by JayjeetAtGithub**
Enhances code quality according to reports from Codacy test

### CODE REVIEW

File: cli/popper/exceptions.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import click
-from click._compat import get_text_stderr
 from click import secho
+from click._compat import get_text_stderr
```
Comment (ivotron) [inline cli/popper/exceptions.py]: another thing we can use to simplify the code is to remove this file (`exceptions.py`) as only `cli.py` uses it to throw a `UsageError` exception, which can be changed for a `ClickException`.

Comment (JayjeetAtGithub) [reply cli/popper/exceptions.py]: @ivotron Sure ! Also this PR is still not ready. I will fix some more issues soon.

**Comment by JayjeetAtGithub** [comment]
@ivotron You may have a look. Thanks.

**Comment by ivotron** [comment]
thanks a lot!

**Pull request merged**